### PR TITLE
Start style guides work

### PR DESCRIPTION
**Pull request by AlexCatch**
This PR includes a slight restructure and the beginning work for style guides, currently only a core JS style guide has been added for the JS side & and the current PHP standards have been moved over.

This PR contains the following changes.

- Moved devops out of the platforms folder, it's not a platform and deserves to be alongside tools at the top level of the repo.
- Moved core php standards from the web folder to the style guides folder, removed the references to Laravel (should be in it's own style guide specific to Laravel) & REST APIs (again, should be in it's own folder specific to REST development as it's not linked to PHP)
- Added the Core JS style guide, this is irrespective of front end framework. It's a fork of the Airbnb JS style guide with a few things removed and everything in it is common sense.
- Corrected links throughout documentation to point to the new places. (Although I might have missed one or two)
- Change `master` to `develop` on our git workflow as per our Slack chats earlier.

### CODE REVIEW

File: styleguides/php/core.md [hunk 1]
```diff
@@ -23,14 +19,6 @@ The below documentation links should be followed unless defined otherwise in thi
 
    * [PSR-1](https://www.php-fig.org/psr/psr-1/)
    * [PSR-2](https://www.php-fig.org/psr/psr-2/)
-   
-#### Laravel
```
Comment (dextermb) [inline styleguides/php/core.md]: Before removing the Laravel docs and the RESTful API standards these should be moved somewhere else

**Comment by AlexCatch** [comment]
@dextermb, i'm merging from my personal fork, as far as I'm concerned i'm not required to follow any specific branching methodologies in my own repos.

**Comment by wheatleyjj** [comment]
That's a crap ton of JS content - nice one. 

We shouldn't just drop the RESTful standards and Laravel docs however, and as the branch stuff is controversial it should probably be its own PR/issue.

It will take a while for everyone to review those JS standards I'd imagine. Potentially only writing down the differences from the AirBnb one (Which people might already know) - could make this an easier read.

**Comment by AlexCatch** [comment]
@dextermb Also the link inside of the PHP standards document is just to the Laravel documentation, it's got no real bearing on PHP standards or Laravel standards for that matter so doesn't belong inside of that file as it's already linked inside the web document.

**Comment by AlexCatch** [comment]
Closed, some more work to be done before a merge.

**Pull request closed without merging**